src/derivatives/05-diffusion-derivatives.md

```diff
@@ -393,7 +394,8 @@ Dictionary `"ResponseFunction"` has the following reserved keywords:
                 "FitMethod": "ols",
                 "OutlierRejectionmethod": "None"
             }
-        }
+        },
+        "ParameterURL": "https://doi.org/10.1002/nbm.1940080707"
```

Comment:
Small quibble, but maybe this one is a better reference? 

```suggestion
        "ParameterURL": "https://doi.org/10.1016/j.jmr.2011.09.022"
```

I believe it's the first reference to the 0-1 _normalized_ FA

Reply:
I'd always had it in my mind that the right citation for FA was the 1996 one. I was just looking through my Zotero and saw the 1995 one, questioned it, found this, and thought maybe all my citations of the 1996 one were erroneous:

![FA_1995](https://github.com/bids-standard/bids-bep016/assets/5637955/b11dedb8-2cea-4556-913d-ce96efe72da7)

I had not realised that it's different to the 1996 one:

![FA_1996](https://github.com/bids-standard/bids-bep016/assets/5637955/9929d9ba-f2db-496a-ba6e-d8c3490ca347)

Is the first one then a mathematical error?